src/common/style/vanilla/navigation.scss

```diff
@@ -60,15 +73,55 @@ $color-navigation-background: #252525;
   // right aligned sign-in navigation
   .p-navigation__links--right {
     @extend .p-navigation__links;
+    margin-right: 20px;
+
+    i {
+      margin-right: .5rem;
+    }
 
     &:last-of-type {
       border-right: none;
     }
+
+    .p-dropdown__toggle {
+      cursor: pointer;
+
+      i {
```

Comment:
This as well (similar to `i` above) - not sure if that's the same icon, or different one. Ideally - class name selector to make it clear.

Reply:
done as above